### PR TITLE
Batch Management SDK 11.0.0

### PR DESCRIPTION
**Pull request by jafreck**

### CODE REVIEW

File: sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs [hunk 1]
```diff
@@ -118,8 +119,8 @@ public async Task BatchPoolEndToEndAsync()
                     // Verify get operation
                     var pool = await this.BatchManagementClient.Pool.GetAsync(resourceGroupName, batchAccountName, iaasPoolName);
                     Assert.Equal("STANDARD_A1", pool.VmSize);
-                    Assert.Null(pool.DisplayName);
-                    Assert.Equal(AllocationState.Steady, pool.AllocationState);
+                    Assert.Equal(displayName, pool.DisplayName);
+                    Assert.Equal(AllocationState.Resizing, pool.AllocationState);
```
Comment (jafreck) [inline sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs:123]: This has always been the expected behavior, however, the transition to `Steady` from `Resizing` is now more latent, causing the faulty assert to fail consistently.

File: sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs [hunk 2]
```diff
@@ -118,8 +119,8 @@ public async Task BatchPoolEndToEndAsync()
                     // Verify get operation
                     var pool = await this.BatchManagementClient.Pool.GetAsync(resourceGroupName, batchAccountName, iaasPoolName);
                     Assert.Equal("STANDARD_A1", pool.VmSize);
-                    Assert.Null(pool.DisplayName);
-                    Assert.Equal(AllocationState.Steady, pool.AllocationState);
+                    Assert.Equal(displayName, pool.DisplayName);
```
Comment (jafreck) [inline sdk/batch/Microsoft.Azure.Management.Batch/tests/ScenarioTests/PoolTests.ScenarioTests.cs:122]: That `displayName` was previously not respected was a bug. This was fixed in the latest release of the Service. It is not called out in the changelog as this bug was fixed for all REST API versions.

File: sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/BatchAccountIdentity.cs [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Batch.Models
+{
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// The identity of the Batch account, if configured. This is only used
+    /// when the user specifies 'Azure.KeyVault' as their Batch account
```
Comment (jafreck) [inline sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/Models/BatchAccountIdentity.cs:18]: This is a typo in the Swagger specification. This should read `Microsoft.KeyVault`. The `EncryptionProperties` object does not have an enum value of `Azure.KeyVault`.

File: sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md [hunk 4]
```diff
@@ -1,10 +1,19 @@
...
+### REST API version
+- This version targets REST API version 2020-05-01.
+
+### Features
+- Added `BatchAccountIdentity` property on `BatchAccount` for enabling system assigned identity when `Microsoft.KeyVault` is specified as the `BatchAccount` encryption property.
+
+### Bug Fixes
+- **[Breaking]** Convert the `PrivateEndpointConnection` update operation to a long running operation.
 
 ## 10.0.0 (2020-04-11)
```
Comment (bgklein) [inline sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md:13]: maybe put deprecated next to the version.

**Comment by isra-fel** [comment]
Hi @jafreck  please leave a link to you openapi spec review PR here. Thanks

File: sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md (hunk 4 above)
Comment (jafreck) [reply sdk/batch/Microsoft.Azure.Management.Batch/CHANGELOG.md:13]: It's technically not deprecated and won't be for at least a year after release. I think a warning that it is not recommended, and that there are known issues should be sufficient.

**Comment by jafreck** [comment]
The REST API PR was not opened through the openapi hub. This is the PR that was open manually: https://github.com/Azure/azure-rest-api-specs/pull/9452

File: sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,5 @@
 # Release History
 
-## 13.1.0-preview.1 (Unreleased)
```
Comment (jafreck) [inline sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md:3]: Should not have been removed

File: sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/BatchManagementClient.cs [hunk 6]
```diff
@@ -368,7 +368,7 @@ private void Initialize()
             PrivateEndpointConnection = new PrivateEndpointConnectionOperations(this);
             Pool = new PoolOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2020-03-01";
+            ApiVersion = "2020-05-01";
```
Comment (isra-fel) [inline sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/BatchManagementClient.cs:371]: Please also update `/sdk/batch/Microsoft.Azure.Management.Batch/AzSdk.RP.props` when you generate from a new API version

Comment (jafreck) [reply sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/BatchManagementClient.cs:371]: Updated this and the assembly version in `AssemblyInfo.cs`.

**Review by bgklein — APPROVED** [review]

**Comment by isra-fel** [comment]
LGTM!

**Pull request merged**